python/lsst/ip/isr/isrTask.py

```diff
@@ -2271,6 +2274,10 @@ def overscanCorrection(self, ccdExposure, amp):
         ampName = amp.getName()
 
         keyBase = "LSST ISR OVERSCAN"
+        # The overscan statistics units will always match the units of
+        # the image at the point they are calculated.
+        metadata[f"{keyBase} SERIAL UNITS"] = metadata.get("LSST ISR UNITS")
+        metadata[f"{keyBase} PARALLEL UNITS"] = metadata.get("LSST ISR UNITS")
```

Comment:
Should this be checking if serial/parallel overscan is turned on?  Especially for cameras that don't have parallel overscan.